source/renderer/app/components/wallet/wallet-restore/ConfigurationDialog.scss

```diff
@@ -32,10 +32,10 @@
       }
     }
 
-    :global
-      .ConfigurationDialog_tooltip.SimpleTooltip_root.TooltipOverrides_root.InlineEditingDropdown-tooltip_root.SimpleTooltip_isShowingOnHover {
+    .tooltip {
       height: 12px;
-      left: -154px;
+      left: -148px;
+      position: relative;
       top: 1px;
       width: 12px;
     }
```

Comment:
Same here, would prefer to ged rid of any css overrides

Reply:
Fixed